src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -168,45 +155,114 @@ protected override
         }
 
         /// <summary>
-        ///
+        /// StopProcessing override
         /// </summary>
         protected override void StopProcessing()
         {
             base.StopProcessing();
             _serializer.Stop();
         }
 
-        #endregion Overrides
+        #endregion PSCmdlet Overrides
 
-        #region file
+        #region IDisposable Members
 
         /// <summary>
-        /// handle to file stream
+        /// Set to true when object is disposed
         /// </summary>
-        private FileStream _fs;
+        private bool _disposed;
 
         /// <summary>
-        /// stream writer used to write to file
+        /// public dispose method
         /// </summary>
-        private XmlWriter _xw;
+        public void Dispose()
+        {
+            if (_disposed == false)
+            {
+                CleanUp();
+            }
+            _disposed = true;
+        }
+
+        #endregion IDisposable Members
+    }
+
+    /// <summary>
+    /// implementation for the Export-Clixml command
+    /// </summary>
+    [Cmdlet(VerbsData.Export, "Clixml", SupportsShouldProcess = true, DefaultParameterSetName = "ByPath", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113297")]
+    public sealed class ExportClixmlCommand : ToClixmlCommand
+    {
+        #region Command Line Parameters
+
+        // If a Passthru parameter is added, the SupportsShouldProcess
+        // implementation will need to be modified.
 
         /// <summary>
-        /// Serializer used for serialization
+        /// mandatory file name to write to
         /// </summary>
-        private Serializer _serializer;
+        [Parameter(Mandatory = true, Position = 0, ParameterSetName = "ByPath")]
+        public string Path { get; set; }
+
+        /// <summary>
+        /// mandatory file name to write to
+        /// </summary>
+        [Parameter(Mandatory = true, ParameterSetName = "ByLiteralPath")]
+        [Alias("PSPath")]
+        public string LiteralPath
+        {
+            get
+            {
+                return Path;
+            }
+            set
+            {
+                Path = value;
+                _isLiteralPath = true;
+            }
+        }
+        private bool _isLiteralPath = false;
+
+        /// <summary>
+        /// Property that sets force parameter.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter Force { get; set; }
+
+        /// <summary>
+        /// Property that prevents file overwrite.
+        /// </summary>
+        [Parameter]
+        [Alias("NoOverwrite")]
+        public SwitchParameter NoClobber { get; set; }
+
+        #endregion Command Line Parameters
+
+        #region FileStream implementation for ToClixmlCommand
+
+        /// <summary>
+        /// handle to file stream
```

Comment:
Please remove the obvious comment . And why the comment is public for private field? Below too.